### PR TITLE
new GRIC terms added in the new file oeso-gric.owl and removed from oeso.owl

### PR DESCRIPTION
**Pull request by pierreetiennealary**

### CODE REVIEW

File: oeso-gric.owl [hunk 1]
```diff
@@ -0,0 +1,2564 @@
...
+        <rdfs:label xml:lang="fr">objectif</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.opensilex.org/vocabulary/oeso#LevelMeasurementRainGauge -->
+
+    <owl:Class rdf:about="http://www.opensilex.org/vocabulary/oeso#LevelMeasurementRainGauge">
+        <rdfs:subClassOf rdf:resource="http://www.opensilex.org/vocabulary/oeso#PrecipitationSensor"/>
+        <dc:source>skos:closeMatch Level measurement rain gauge (LRG) [IOM-99_FI-RI 2.2.1e] http://www.wmo.int/pages/prog/www/IMOP/publications/IOM-99_FI-RI.pdf</dc:sour
```
Comment (annetireau) [inline oeso-gric.owl]:
```suggestion
    <owl:Class rdf:about="http://www.opensilex.org/vocabulary/oeso#Grain">
```

Comment (annetireau) [reply oeso-gric.owl]: What is the difference between grain and kernel ?

Comment (annetireau) [inline oeso-gric.owl]:
```suggestion
    <owl:Class rdf:about="http://www.opensilex.org/vocabulary/oeso#Spikelet">
```

Comment (annetireau) [inline oeso-gric.owl]:
```suggestion
    <owl:Class rdf:about="http://www.opensilex.org/vocabulary/oeso#Stem">
```

Comment (annetireau) [inline oeso-gric.owl]:
```suggestion
    <owl:Class rdf:about="http://www.opensilex.org/vocabulary/oeso#Tassel">
```

Comment (annetireau) [inline oeso-gric.owl]:
```suggestion
    <owl:Class rdf:about="http://www.opensilex.org/vocabulary/oeso#Tiller">
```

**Review by annetireau — CHANGES_REQUESTED** [review]

File: oeso-gric.owl (hunk 1 above)
Comment (annetireau) [inline oeso-gric.owl]: Are there 2 labels in english for grain ? grain and kernel ?

**Review by annetireau — CHANGES_REQUESTED** [review]

File: oeso-gric.owl (hunk 1 above)
Comment (pierreetiennealary) [reply oeso-gric.owl]: yes, for wheat either grain or kernel can be used

Comment (pierreetiennealary) [reply oeso-gric.owl]: we distinguish the common term "seed" (PO:0025496) and the more specific term "grain" (PO:0030104) used for Poacea and synonym to caryopsis fruit and kernel. See http://agrovoc.uniroma2.it/agrovoc/agrovoc/en/page/?uri=http%3A%2F%2Faims.fao.org%2Faos%2Fagrovoc%2Fc_25387

Comment (annetireau) [reply oeso-gric.owl]: Maybe you should create 2 concepts or keep the less specific

Comment (pierreetiennealary) [reply oeso-gric.owl]: I'll remove the "kernel" and "caryopisi fruit" as pseudo synonyms and replace the "grain" label by "caryopsis fruit" in order to keep the PO term

**Review by annetireau — CHANGES_REQUESTED** [review]
Please don't commit conflics on oeso-phenoarch.owl

**Pull request closed without merging**